docs/details/signal.dox

```diff
@@ -211,54 +211,109 @@ respectively, then the possible batch operations are as follows.
 \defgroup signal_func_approx1 approx1
 \ingroup approx_mat
 
-approx1 interpolates data along a single dimension.
+approx1 interpolates data along one dimension. Interpolation computes
+for values of unknown points out of known ones.
```

Comment:
I agree with Pradeep on moving the second sentence to the next paragraph. That way too you can expand more on what interpolation means and not be restricted to keeping it short because it's part of the brief description. I mean, for me at least, the way you described interpolation works for extrapolation as well, so I think it needs a little more detail and so I think it should be moved after the first line/brief description